amethyst_renderer/src/pipe/pass.rs

```diff
@@ -158,11 +158,21 @@ impl<P> CompiledPass<P> {
 
     pub fn new_target(&mut self, target: &Target) {
         for effect in &mut self.effects {
+            // Distribute new no blend targets
```

Comment:
This comment confused me for a couple of seconds. I don't know how to improve it though.

Reply:
How about "new targets that don't blend"?